userdocs/profiles.dev/docs/installer-docs/installing-via-gitops.md

```diff
@@ -22,6 +22,28 @@ The full setup docs can be found [here](/docs/tutorial-basics/setup#prerequisite
 
 ## Simple install from a profile URL
 
+
+### Bootstrapping your local git repository
+You can use the `flux bootstrap` command to save commonly used `pctl` configuration to your GitOps repo.
+
+Once such piece of configuration is your repo's `GitRepository` resource.
```

Comment:
```suggestion
Once such piece of configuration is your flux repo's `GitRepository` resource.
```

Reply:
Maybe `GitOps` repo to be consistent. we have not called it Flux repo anywhere else